README.md

```diff
@@ -316,6 +359,53 @@ style: |
   }
 ```
 
+Change the title size 
+```yaml
+    style: |
+      .title {
+        font-size: var(--ha-font-size-l);
+      }
+```
+
+## Card Mod
+
+Card mod still works, but it's recommended to use the `style` attribute directly for better integration.
+
+With the help of the integration [card mod](https://github.com/thomasloven/lovelace-card-mod), the card can be flexibly adapted. This is also possible based on the card status. A CSS class “open” or “close” is always set.
+
```

Comment:
I don’t think we need card-mod examples any more. These are all in styling examples. The only element which is in a shadow root in the svg for the icon and I don’t know of any reason you would need to style it. So good to mention card-mod but remove examples in my opinion.

Reply:
@dcapslock Agree, i replaced the whole chapter. I think should be clear for everyone :-)